packages/components/src/components/checkbox/checkbox.css

```diff
@@ -215,17 +215,49 @@ scale-checkbox.indeterminate [part='container']:active [part='checkbox'] {
   border-color: var(--telekom-color-primary-pressed);
 }
 
+scale-checkbox:not(.checked) [part='container']:active [part='checkbox'] {
+  background: var(--telekom-color-ui-outlined-fill-pressed);
+  /* border-color: var(--telekom-color-ui-outlined-border-pressed); */
```

Comment:
hmm, why this line is commented out and the next line is exact the same? Could it be that you forgot to remove it?